### PR TITLE
[Quorum] added molecule ambassador-certs scenario

### PR DESCRIPTION
**Pull request by angelaalagbe**
Signed-off-by: Angela Alagbe <angela.alagbe@accenture.com>

**Changelog**
- Added a molecule ambassador-certs test scenario
- Updated ambassador-certs nested_main.yaml file

 

**Reviewed by**
@sownak 
 

**Linked issue**
#719

### CODE REVIEW

File: platforms/quorum/configuration/molecule/ambassador-certs/converge.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    - name: "Copy openssl.conf to molecule directory"                 # Copying openssl.conf to molecule playbook directory
+      copy:
+        src: "{{ playbook_dir }}/../../openssl.conf"
+        dest: "{{ playbook_dir }}"
+        
+    - name: "Test ambassador-certs role"
+      include_role:
+        name: "configuration/roles/create/certificates/ambassador"
+      vars:
+        auth_lists:
```
Comment (sownak) [inline platforms/quorum/configuration/molecule/ambassador-certs/converge.yml:106]: all of these variables are not needed. Please check the role `create/certificates/ambassador` and remove the vars that are not used

File: platforms/quorum/configuration/molecule/default/INSTALL.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
+*******
+Docker driver installation guide
+*******
+
+Requirements
+============
+
+* Docker Engine
+
+Install
```
Comment (sownak) [inline platforms/quorum/configuration/molecule/default/INSTALL.rst:10]: this file is not needed

File: platforms/quorum/configuration/molecule/default/converge.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Converge
+  hosts: all
```
Comment (sownak) [inline platforms/quorum/configuration/molecule/default/converge.yml:3]: The default role is empty

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/quorum/configuration/molecule/default/converge.yml (hunk 3 above)
Comment (angelaalagbe) [reply platforms/quorum/configuration/molecule/default/converge.yml:3]: Dimbu is working on this

**Review by sownak — APPROVED** [review]

**Pull request merged**